lib/tacos/path_filter.py

```diff
@@ -24,30 +25,43 @@ def match(self, path: str) -> bool:
         return False
 
     @classmethod
-    def from_config(cls, path: OSPath = CONFIG_PATH) -> PathFilter:
+    def from_config(cls, configs: list[str]) -> PathFilter:
         """Get a list of allowed globs from a file
 
         Hash comments are removed and blank lines are ignored.
         Inline comments are not allowed
-        An empty or missing file is treated as all allowed.
+        If both the provided and default files are missing, allow all.
         Globs are evaluated with the fnmatch module."""
         lines: list[str] = []
-        # remove from the first unescaped # to the end
-        try:
-            with path.open() as config:
-                for line in config:
-                    line = line.strip()
-                    if line and not line.startswith("#"):
-                        lines.append(line)
-        except FileNotFoundError:
-            pass
+        for path in [OSPath(x) for x in configs] + [OSPath(DEFAULT_PATH)]:
+            try:
+                with path.open() as config_file:
+                    sh.info(
+                        f"TACOS-gha: Using slice-allowlist from path: {path}"
+                    )
+                    for line in config_file:
+                        line = line.strip()
+                        if line and not line.startswith("#"):
+                            lines.append(line)
+                break
```

Comment:
This is only making use of the first config file even if the user passed two? I would normally expect the system to concatenate the configs in this case.

If you treat a missing file as equal to an empty file I believe you get the same, correct behavior but with a bit fewer edge cases to consider.